client/src/profile/Addresses.js

```diff
@@ -0,0 +1,35 @@
+import React, { PropTypes } from 'react';
+import AddressField from './AddressField';
+
+const Addresses = ({
+  addresses,
+  saveNewAddress,
+  updateAddressState,
+  addressFieldState,
+  removeAddress,
+ }) => (
+   <div>
+     <h3>Receive Alerts For Which Addresses?</h3>
+     {addresses.map(address => (
+       <div key={address.address}>
```

Comment:
I try to keep them around 1x of my screen height but any time you have a chunk of functionality/use it's a good opp to pull it out. It's generally considered to be preferable to make components as small as possible.